aspnetcore/tutorials/publish-to-iis.md

```diff
@@ -62,7 +62,7 @@ Download the installer using the following link:
 
 ## Create an ASP.NET Core Razor Pages app
```

Comment:
Ideally, we want this to not be Razor Pages. Maybe we make this section generic and just say to create an ASP.NET Core app without saying what kind?